echo_filter.py

```diff
@@ -56,6 +59,18 @@ def generic_question(update: Update, context: CallbackContext) -> bool:
     return False
 
 
+def repeat_word_sarcastically_filter(update: Update, context: CallbackContext) -> bool:
```

Comment:
we should decide if we want to use type hinting in function deceleration.
what do you think?

Reply:
python typing sucks really hard and i find it super annoying to keep track of. you can change it if you want